### PR TITLE
hovering effect added on recepies

### LINKED ISSUE #46: Hovering on items
image
Hey @AlfiyaSiddique, the hover effect is currently missing on items, which affects the user experience. I can implement it to enhance the overall interactivity. Please assign this issue to me under the gssoc-ext label.

### PR DESCRIPTION
**Pull request by INam1995**
## What does this PR do?

it will add hovering effect on items
@AlfiyaSiddique Kindly merge it if there is an issue let me know
Please don't forget to add label

Fixes #46 

## How should this be tested?

videos is provided here for testing and reference

- Test 1
- Test 2

https://github.com/user-attachments/assets/237120ba-15b8-438f-b184-b81dab6b7582

### Required

- [ x] Filled out the "How to test" section in this PR
- [x ] Read Contributing guide
- [x ] Self-reviewed my own code
- [ x] Checked for warnings, there are none
- [ x] Removed all `console.logs`
- [ x] The changes don't cause any responsiveness issues

### CODE REVIEW

**Comment by AlfiyaSiddique** [comment]
@INam1995 the deployment is failing due to the below issue:
image

Please fix it

**Comment by INam1995** [comment]
@AlfiyaSiddique can you please specify in which file it's giving error? i checked all recepie-related files, but i didn't get 
So please help me by specifying the file name so that I can proceed asap!!

**Comment by INam1995** [comment]
@AlfiyaSiddique please let me know so that i can proceed asap!!

**Comment by AlfiyaSiddique** [comment]
File path: : frontend/src/Components/Cards

**Comment by INam1995** [comment]
@AlfiyaSiddique done please check it

File: backend/server.js [hunk 1]
```diff
@@ -21,6 +21,25 @@ const allowedOrigins = [
...
+//             } else {
+//                 // Provide a more informative error message if necessary
+//                 callback(new Error("CORS policy: This origin is not allowed."));
+//             }
+//         },
+//     })
+// );
+
+app.use(cors({origin:"http://localhost:5173"})) // for local use
+
```
Comment (AlfiyaSiddique) [inline backend/server.js:42]: @INam1995 please active the commented code and comment out the local code

File: package.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+  "version": "1.0.0",
+  "description": "**TastyTrails** is more than just recipes — it’s a vibrant social network where food lovers can connect, discover, and share the unique flavors of their cultures. A platform for passionate food enthusiasts to dive into a world of tastes!",
+  "main": "index.js",
+  "scripts": {
+    "start": "concurrently \"npm run dev --prefix backend\" \"npm run dev --prefix frontend\""
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
   "devDependencies": {
```
Comment (AlfiyaSiddique) [inline package.json:12]: @INam1995 did you write this or autogenerated? If you write then please remove

Comment (INam1995) [reply package.json:12]: done @AlfiyaSiddique

File: backend/server.js (hunk 1 above)
Comment (INam1995) [reply backend/server.js:42]: > @INam1995 please active the commented code and comment out the local code

done @AlfiyaSiddique

File: package.json (hunk 2 above)
Comment (INam1995) [reply package.json:12]: > @INam1995 did you write this or autogenerated? If you write then please remove

@AlfiyaSiddique it's autogenerated

**Comment by INam1995** [comment]
@AlfiyaSiddique is there any issue?

**Review by AlfiyaSiddique — APPROVED** [review]

**Pull request merged**